scientific_library/tvb/simulator/models/__init__.py

```diff
@@ -92,6 +92,10 @@ def _get_imported_model(model):
 
 
 _module_models = {
+    'reducedwongwangt': [ModelsEnum.REDUCEDWONGWANGT],
+    'reducedwongwang': [ModelsEnum.REDUCEDWONGWANG],
+    'montbrio': [ModelsEnum.MONTBRIO],
+    'generic2doscillator': [ModelsEnum.GENERIC2DOSCILLATOR],
```

Comment:
These new models won't work for us. 
Neither the modules nor the enum entries exist.

Reply:
The enum will be generated as well when you run RateML (new DSL name). This makes sure that TVB recognizes the newly generate model. This is the file not necessary needs to be committed and I resetted it, but lines needs to be added during runtime. The _find_lems_models is not my work. For me it worked fine without this function.

Reply:
OK. Then I will try to remove the function as part of another review PR.
Thanks for resetting the dictionary.